apps/sample-app/README.md

```diff
@@ -1,11 +1,61 @@
 # Sample App
 
-Sample app leveraging teams js v2 for multi host experience
+The sample app leverages TeamsJS v2 SDK for creating a multi host app experience. The sample app displays the usage of the following TeamsJS v2 APIs:
 
-(I will be working on this section below in the coming few weeks)
+- core.openLink
+- authentication.authenticate
+- isSupported
+- app.getContext
+- app.initialize
+- app.registerOnThemeChangeHandler
+- call.startCall
+- chat.OpenSingleChatRequest
+- mail.openMailItem
+- calendar.composeMeeting
+- pages.navigatetoApp
+- pages.shareDeepLink
 
-## Getting Started
+## Running the Sample App Online
 
-### Running the Sample App
+### Set up the Sample App
 
-## Troubleshooting
+1. Clone the TeamsJS v2 repository on your local machine using the following command in the terminal:
+   `git clone https://github.com/OfficeDev/microsoft-teams-library-js.git`
+2. Run `yarn install` from the repository root
+3. Run `yarn build` form repository root
+4. Change directory to the sample app by running the following command
+   `cd apps/sample-app`
+5. Run `yarn start`
+   - Open https://localhost:4003 window in your browser.
+   - Proceed to 'continue to localhost' upon receiving 'Your Connection isn't private' warning.
+   - An alert will pop up stating the app must be sideloaded onto a host (Teams).
+   - Please follow the steps below to sideload the app onto Teams.
+
+### Sideload the App to Teams Web
+
+6. Open Microsoft Teams Web
+   - Click '+ Apps'>Manage Your Apps> Upload An App> Upload Custom App
+   - You have to upload sample-app-zipped.zip file here. This file can be found in your local copy of microsoft-teams-library-js from step 1 and head to apps -> sample-app -> sample-app-zipped.zip
+   - After uploading sample-app-zipped.zip, click ‘+Add’
+
+The app will start running on Teams and will automatically sideload onto Outlook and Office Web.
+
+## Running the Sample App on Desktop Applications
+
+To have the app run on the desktop applications of Teams, Outlook and Office, an SSL certificate will have to be created.
+
+1. In the sample-app folder, run
+   `npx mkcert create-ca`
+   `npx mkcert create-cert`
+2. Edit 'devServer' in the webpack.config.js folder inside sample-app to point at the following files this way -
+   `devServer: {`
+   `// ...`
+   `https: {`
+   `key: fs.readFileSync("cert.key"),`
+   `cert: fs.readFileSync("cert.crt"),`
+   `ca: fs.readFileSync("ca.crt"),`
+   `},`
+   `// ....`
+   `},`
```

Comment:
You can do multi-line code snippets with triple backticks, like this:

\`\`\`
// code snippet
\`\`\`